misc/sample_application/uservm/ivshmemlib.c

```diff
@@ -0,0 +1,148 @@
+/*
+* Copyright (C) 2022 Intel Corporation.
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#include "ivshmemlib.h"
+
+//The shared memory region that is used for inter-vm shared memory
+char *ivshmem_ptr = NULL;
+
+/*
+int setup_ivshmem_region(const char *f_path)
```

Comment:
I'm thinking it may be good to use Doxygen here especially if we intent to use this as an illustration of how to use the `ivshmem` mechanis. Thoughts @dbkinder @amyreye ?